packages/firestore/src/local/shared_client_state.ts

```diff
@@ -676,9 +682,13 @@ export class WebStorageSharedClientState implements SharedClientState {
       }
     }
 
-    this.localClientState.addQueryTarget(targetId);
-    this.persistClientState();
+    // If the query is listening to cache only, the target ID should not be registered the  with
```

Comment:
A couple of issues with articles here:

"If the query is listening to cache only, the target ID should not be registered with the local Firestore client as an active watch target."

At least, I *think* the "an" is required.  WDYT Mark or others?